### PR TITLE
CGT-1307 - Added investments view content and validation

### PR DESCRIPTION
**Pull request by macgrewal**

### CODE REVIEW

File: test/common/ValidationSpec.scala [hunk 1]
```diff
@@ -120,4 +120,109 @@ class ValidationSpec extends UnitSpec {
...
-}
+
+  "calling bigDecimalCheck" when {
+
+    "input contains non-numeric characters" should {
+      "fail" in {
+        bigDecimalCheck("abc") shouldBe false
+      }
+    }
+
```
Comment (DavidWoodward37) [inline test/common/ValidationSpec.scala]: Do you want a validation check to also check for the "" => true condition.

File: test/forms/resident/ImprovementsFormSpec.scala [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+        form.value.get shouldBe ImprovementsModel(BigDecimal(1))
+      }
+    }
+
+    "supplied with valid amount" should {
+      "not raise form error" in {
+        val form = improvementsForm.bind(Map("amount" -> "1"))
+        form.hasErrors shouldBe false
+      }
+    }
```
Comment (DavidWoodward37) [inline test/forms/resident/ImprovementsFormSpec.scala]: Could you refactor the two above tests into the same "supplied with a valid amount" should "...." in {.....} "....." in {....} then you could also use the val of the form for both?

File: test/assets/MessageLookup.scala [hunk 3]
```diff
@@ -82,8 +82,11 @@ object MessageLookup {
   }
 
   //Improvements messages
-  object improvements {
+  object improvementsView {
```
Comment (DavidWoodward37) [inline test/assets/MessageLookup.scala]: Should we change the other objects to this naming style too?

Comment (macgrewal) [inline test/assets/MessageLookup.scala]: changed to stop scala-style whingeing. We could do this on other views also.

**Pull request merged**